psi4/src/psi4/libpsipcm/psipcm.cc

```diff
@@ -159,6 +160,7 @@ PCM::PCM(const std::string &pcmsolver_parsed_fname, int print_level, std::shared
     std::vector<int> tmp(int(ntess_ / ntessirr_));
     std::fill(tmp.begin(), tmp.end(), ntessirr_);
     tesspi_ = Dimension(tmp);
+    set_dynamic_asc(false);
```

Comment:
One should set whether to use the static or dynamic (aka optical) permittivity in the PCM input itself using the `nonequilibrium` keyword:
```
pcm = {
   Units=Angstrom
   Medium {
   SolverType = IEFPCM
   Solvent = Water
   Nonequilibrium = True
   }

   Cavity {
   Type = GePol
   Area = 0.3
   Mode = Implicit
   }
}
```
then you can invoke just `pcmsolver_compute_response_asc` in the `compute_Ve_PCM` function and that will do the right thing, _i.e._ use the static permittivity if `Nonequilibrium = False` or the dynamic otherwise.